src/api/routers/participants/participantsKeyPairs.ts

```diff
@@ -1,12 +1,13 @@
 import { Response } from 'express';
 
+import { siteIdNotSetError } from '../../helpers/errorHelpers';
 import { getKeyPairsList } from '../../services/adminServiceClient';
 import { ParticipantRequest } from '../../services/participantsService';
 
 export async function getParticipantKeyPairs(req: ParticipantRequest, res: Response) {
   const { participant } = req;
   if (!participant?.siteId) {
-    return res.status(400).send('Site id is not set');
```

Comment:
So many magic strings :( this also means our error code for this error is in one place.